src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -10,13 +10,18 @@
 public final class GATKVCFConstants {
```

Comment:
Can you check the contents of our `GATKVCFConstants` against the GATK3 version? They should be identical at this point, I would think.